### ISSUE TITLE
Fake-Menu-Button: items in story are missing their href attributes

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1563]
Seems like the anchors are now missing their href attributes, which makes it seem like the example is not keyboard accessible.

![Screen Shot 2021-11-29 at 1 19 28 PM](https://user-images.githubusercontent.com/38065/143944588-6dbfa007-e043-4f0f-91f6-c698bedadbf3.png)

https://ebay.github.io/ebayui-core/?path=/story/ebay-fake-menu-button--standard
